catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -223,6 +236,8 @@ public Metacard transform(InputStream input, String id)
             }
 
             String metacardContentType = metacard.getContentTypeName();
+            metacard.setAttribute(new AttributeImpl(Core.DATATYPE, getDatatype(metacardContentType)));
```

Comment:
Most (if not all) InputTransformers should know the Mime-type (e.g. CSW knows it handles application/xml although it probably hasn't been updated to set that attribute, maybe all existing transformers need to be updated?).